nixos/modules/services/mail/dovecot.nix

```
@@ -106,6 +119,63 @@ let
     }) cfg.sieve.pipeBins
   );
 
+  ### >=2.4 config
+
+  yesOrNo = v: if v then "yes" else "no";
+
+  toOption =
+    i: n: v:
+    "${i}${toString n} = ${v}";
+
+  parseSection =
```

```suggestion
  formatKeyValue =
```

The name is completely different from what this actually does.